pages/docs/linkfree-profile-tips.mdx

```diff
@@ -0,0 +1,21 @@
+import DocsLayout from "@components/layouts/DocsLayout.js"; import Link from "@components/Link";
+
+# Linkfree Profile Tips 
```

Comment:
All our doc pages start with the 2nd heading for accessibility because I think the docs layout has the first heading, please check with other docs pages so we can be consistent.

Reply:
Noted